<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetPoint</class>
 <widget class="QWidget" name="WidgetPoint">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1097</width>
    <height>567</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Pressure Sensor</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditSensor">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Shaft</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditShaft">
       <property name="text">
        <string/>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonReset">
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCleanUp">
       <property name="text">
        <string>Clean-up...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCompute">
       <property name="text">
        <string>Compute...</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox"/>
     </item>
     <item>
      <widget class="QTableView" name="tableViewBestParams">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>60</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>3</number>
     </property>
     <widget class="QWidget" name="tabInfos">
      <attribute name="title">
       <string>Infos</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,1">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0">
           <item>
            <widget class="QGroupBox" name="groupBox_2">
             <property name="title">
              <string>Notice</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QPlainTextEdit" name="plainTextEditNotice"/>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_3">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="title">
              <string>Infos</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_4">
              <item row="0" column="0">
               <widget class="QTableView" name="tableViewInfos"/>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Installation</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QScrollArea" name="scrollArea">
              <property name="widgetResizable">
               <bool>true</bool>
              </property>
              <widget class="QWidget" name="scrollAreaWidgetContents">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>1029</width>
                 <height>140</height>
                </rect>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_2">
                <item>
                 <widget class="QLabel" name="labelSchema">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text">
                   <string>TextLabel</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabDataArrayAndPlots">
      <attribute name="title">
       <string>Data arrays and plots</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QCheckBox" name="checkBoxRaw_Data">
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="text">
          <string>Show Raw Data</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QScrollArea" name="scrollAreaDataPlot">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_7">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>1055</width>
             <height>155</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <item>
             <widget class="QGroupBox" name="groupBoxPress">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <pointsize>15</pointsize>
               </font>
              </property>
              <property name="title">
               <string>Pressure</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBoxTemp">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <pointsize>15</pointsize>
               </font>
              </property>
              <property name="title">
               <string>Temperatures</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
         <widget class="QTableView" name="tableViewDataArray"/>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabFluxes">
      <attribute name="title">
       <string>Fluxes</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QSplitter" name="splitter_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QScrollArea" name="scrollArea_11">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_16">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>1055</width>
             <height>212</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QSplitter" name="splitter_6">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <widget class="QGroupBox" name="groupBoxWaterFlux">
               <property name="title">
                <string/>
               </property>
              </widget>
              <widget class="QGroupBox" name="groupBoxAdvectiveFlux">
               <property name="contextMenuPolicy">
                <enum>Qt::DefaultContextMenu</enum>
               </property>
               <property name="title">
                <string/>
               </property>
               <property name="flat">
                <bool>false</bool>
               </property>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
         <widget class="QScrollArea" name="scrollArea_14">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_18">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>1055</width>
             <height>212</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QSplitter" name="splitter_5">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <widget class="QGroupBox" name="groupBoxTotalFlux">
               <property name="title">
                <string/>
               </property>
              </widget>
              <widget class="QGroupBox" name="groupBoxConductiveFlux">
               <property name="contextMenuPolicy">
                <enum>Qt::DefaultContextMenu</enum>
               </property>
               <property name="title">
                <string/>
               </property>
               <property name="flat">
                <bool>false</bool>
               </property>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabTemperature">
      <attribute name="title">
       <string>Temperature</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QSplitter" name="splitter_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QScrollArea" name="scrollArea_7">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_10">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>1055</width>
             <height>212</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_9">
            <item>
             <widget class="QSplitter" name="splitter_7">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <widget class="QGroupBox" name="groupBox_6">
               <property name="title">
                <string/>
               </property>
               <widget class="QRadioButton" name="radioButton">
                <property name="geometry">
                 <rect>
                  <x>110</x>
                  <y>50</y>
                  <width>112</width>
                  <height>23</height>
                 </rect>
                </property>
                <property name="text">
                 <string>Display quantile 1</string>
                </property>
               </widget>
               <widget class="QRadioButton" name="radioButton_2">
                <property name="geometry">
                 <rect>
                  <x>200</x>
                  <y>90</y>
                  <width>112</width>
                  <height>23</height>
                 </rect>
                </property>
                <property name="text">
                 <string>Displau quantile 2</string>
                </property>
               </widget>
              </widget>
              <widget class="QGroupBox" name="groupBoxTempMap">
               <property name="contextMenuPolicy">
                <enum>Qt::DefaultContextMenu</enum>
               </property>
               <property name="title">
                <string/>
               </property>
               <property name="flat">
                <bool>false</bool>
               </property>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
         <widget class="QScrollArea" name="scrollArea_8">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_11">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>1055</width>
             <height>212</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QSplitter" name="splitter_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <widget class="QGroupBox" name="groupBoxUmbrella">
               <property name="title">
                <string/>
               </property>
              </widget>
              <widget class="QGroupBox" name="groupBoxTempDepth">
               <property name="title">
                <string/>
               </property>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabMCMC">
      <attribute name="title">
       <string>MCMC Results</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_13" stretch="0,1">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Best Parameters (rounded)  :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Bins :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="horizontalSliderBins">
           <property name="minimum">
            <number>4</number>
           </property>
           <property name="maximum">
            <number>200</number>
           </property>
           <property name="value">
            <number>4</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="invertedAppearance">
            <bool>false</bool>
           </property>
           <property name="invertedControls">
            <bool>false</bool>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksAbove</enum>
           </property>
           <property name="tickInterval">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelBins">
           <property name="text">
            <string>60</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonRefreshBins">
           <property name="text">
            <string>Refresh</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QScrollArea" name="scrollArea_5">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_6">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>1055</width>
            <height>397</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QVBoxLayout" name="vboxhistos"/>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
